src/main/java/com/chargerrobotics/subsystems/DriveSubsystem.java

```diff
@@ -140,7 +276,24 @@ public void tankDrive(double leftPower, double rightPower) {
   }
 
   public void arcadeDrive(double throttle, double turnRate) {
-    differentialDrive.arcadeDrive(throttle, turnRate);
+    if (RobotBase.isReal()) {
+      differentialDrive.arcadeDrive(throttle, turnRate);
+    } else {
+      fakeDifferentialDrive.arcadeDrive(turnRate, throttle); // yes this is intentional
```

Comment:
Not setting `leftDriveGroup.setInverted(true);` for the simulation _might_ be related to why you need this, though can't say for certain.

Reply:
Inverting the `leftDriveGroup` didn't have an effect on the simulation when calling `arcadeDrive`. I feel like at this point in time, it isn't worth investigating in more detail. Currently, the code accurately simulates the robot running in `arcadeDrive`. @skallaher what are your thoughts on this?